src/lib/plugin.js

```diff
@@ -379,8 +379,8 @@ class FiveBellsLedger extends EventEmitter2 {
     if (!res || res.statusCode !== 200) { throwErr() }
 
     // note that the fivebells ledger API uses 'scale' instead of 'currency_scale'.
-    // the 'ilp_prefix' field is not mandatory
-    if (!res.body.currency_code || !res.body.scale) { throwErr() }
+    // fields like 'ilp_prefix' and 'currency_code' are not mandatory
+    if (!res.body.connectors || !res.body.scale) { throwErr() }
```

Comment:
ah right, and I also needed to name that branch `mj-currency_scale` and push it to origin of course, so that [circleci can use it](https://circleci.com/gh/interledgerjs/five-bells-integration-test/407),